### ISSUE TITLE
[feature] dedicated bot timeline

### ISSUE THREAD

**Issue by shleeable** [mastodon/mastodon#8869]
After #8347, I figure I'd love to be able to remove bots from the federated timeline...
and even have a dedicated federated timeline [Bots only]

This kind of thing is rather out of the box, or enabled by custom filters [compute heavy]

<img width="435" alt="image" src="https://user-images.githubusercontent.com/17537000/47256175-43732480-d4c7-11e8-84e2-63f07b002b29.png">

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by ocdtrekkie** [comment]
This is a duplicate of #8293
